rcnn/README.md

```diff
@@ -0,0 +1,71 @@
+# Rcnn
+
+The Pytorch implementation is [facebookresearch/detectron2](https://github.com/facebookresearch/detectron2).
+
+## Models
+
+- [x] Faster R-CNN(R50-C4)
+
+- [ ] Mask R-CNN(R50-C4)
+
```

Comment:
do you mean Faster RCNN not works for now?

Reply:
sorry, i made a mistake.
I mean, faster-rcnn is working